resources/themes/cura-light/theme.json

```diff
@@ -646,7 +646,10 @@
         "wizard_progress": [10.0, 0.0],
 
         "message": [30.0, 5.0],
-        "message_close": [2, 2],
+      "message_close": [
+        2,
+        2
+      ],
```

Comment:
Why change this?

Reply:
Probably a zealous IDE changing the indentation, there are other occurences.